AK/Platform.h

```diff
@@ -7,6 +7,10 @@
 
 #pragma once
 
+#if __has_include(<features.h>)
```

Comment:
just to confirm, `__has_include` is always a thing on both clang and gcc for the versions we care about (i.e. with c++20 support)? we had to ifdef `__has_feature` or some other thing later in this file.

Reply:
https://en.cppreference.com/w/cpp/preprocessor/include says it's in C++17